enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -658,7 +658,7 @@ By default, this component takes full page height (100vh). You can override this
 <EditableDashboard
   style={{
     height: 800,
-    overflow: "auto",
+    minHeight: "auto",
```

Comment:
should we add `style` to `InteractiveDashboard` example as well? I see you also use the same style in your video demo.